src/Refactoring-UI/RBDeprecateClassDriver.class.st

```diff
@@ -80,12 +80,10 @@ RBDeprecateClassDriver >> requestClassToMigrateReferencesTo [
 
 	newClassName := SpSelectDialog new
 		                title: 'Class to migrate ' , className;
-		                label:
-			                'The new class that will be used to migrate all references of the old class to it.';
-		                items: Smalltalk allClassesAndTraits;
+		                label: 'The new class that will be used to migrate all references of the old class to it.';
+		                items: self class environment allClasses; "This should probably check in the environment of the class to deprecate but I don't know how to get it here"
```

Comment:
You can get it with `RBDeprecateClassRefactoring` instance that inherits `#model` from `RBAbsytractTransformation`.
As `RBDeprecateClassRefactoring` is instanciated via new, you will get the defaul model that is: `RBNamespace onEnvironment: RBBrowserEnvironment new`.
So in the end, I'm not sure there is value to get it. But in the case you want to, just instantiate `RBDeprecateClassRefactoring` before running `requestClassToMigrateReferencesTo:` in `requestClassToMigrateReferencesTo`